towhee/functional/mixins/dataframe.py

```diff
@@ -30,15 +30,15 @@ class EntityMixin:
     1. define an operator with `register` decorator
 
     >>> from towhee import register
-    >>> from towhee import DataCollection
+    >>> from towhee import DataFrame
```

Comment:
I think we have to be careful here. DataFrame already exists for the original engine. This could lead to incorrect imports. I think we either should use a different name or use absolute imports of the correct dataframe.

Reply:
Yeah, make sense, I'll discuss with @reiase

Reply:
> I think we have to be careful here. DataFrame already exists for the original engine. This could lead to incorrect imports. I think we either should use a different name or use absolute imports of the correct dataframe.

This DataFrame will be as a part of Towhee API. And the original one is part of internal implementation. We will rename the original one in the next round of engine refactor.